src/app/(main)/dashboard/components/Calendar/Calendar.logic.ts

```diff
@@ -0,0 +1,140 @@
+import dayjs from "dayjs";
```

Comment:
We are using date-fns for this project. I'd rather you stick to using that instead of having multiple date package in the project.

Reply:
Ok, I'm confused, where is dayjs coming from? I see it's not something you installed separately

Reply:
I've used dayjs because it was already in the project, if we don't want to use it, I must rethink almost all the logic of the calendar

Reply:
I saw now that it is a dependency for cypress

Reply:
It is a lightweight dependency, are you sure it is better to rethink all the logic?